crates/base/Cargo.toml

```diff
@@ -6,6 +6,7 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+base_share = { version = "0.1.0", path = "../base_share" }
```

Comment:
Wonder if we should call it `base_share` or can we make it `http_exts` or `http_utils`? Want to keep the crates focused and less confusing. Having a `base_share` crate may cause confusions for other contributors.

Reply:
@laktek 
Yeah, `http_utils` would be better.
I’ll reflect that 😁

Reply:
done